live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -15,9 +15,16 @@
 #
 
 ---
+#
+# Install the virtualization package. This package needs to configure nscd,
+# which might not be installed yet, so we make sure nscd is installed first.
+#
 - apt:
-    name: delphix-virtualization
+    name: "{{ item }}"
     state: present
+  with_items:
+    - nscd
```

Comment:
Why do you list `nscd` here, rather than make it a dependency of the `delphix-virtualization` package?